apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const router = express.Router();
+const {loadSqlEquiv} = require('@prairielearn/postgres');
+const sqldb = require('@prairielearn/postgres');
+const sql = loadSqlEquiv(__filename);
+
+router.use((req, res, next) => {
+  // Initialize policies array in res.locals
+  res.locals.policies = []; 
+  next();
+});
+
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      current_student_uid: res.locals.user.uid,
+      current_assessment_title : res.locals.assessment.title
+    };
+    const result = await sqldb.queryAsync(sql.selectQuery, params);
+    res.locals.policies = result.rows
+    // console.log(res)
+    res.render(__filename.replace(/\.js$/, '.ejs'), {
+      policies: res.locals.policies,
+      assessment_id: params.assessment_id,
+      current_student_uid: params.current_student_uid,
+      current_assessment_title: params.current_assessment_title
+    });
+  })
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (req.body.__action === 'add_new_override') {
+    const params = {
+      assessment_id: req.body.assessment_id,
```

Comment:
I'd recommend pulling from `res.locals.assessment.id` instead. As written, this would allow an instructor to create overrides for assessments that are in other people's course because there's no validation being done for `req.body.assessment_id`.